src/sentry/static/sentry/app/views/usageStats/usageStatsOrg.tsx

```diff
@@ -327,7 +327,9 @@ class UsageStatsOrganization extends AsyncComponent<Props, State> {
       return (
         <Panel>
           <PanelBody>
-            <LoadingIndicator />
+            <LoaderWrapper>
+              <LoadingIndicator />
```

Comment:
I wonder if we should be using the content block style loading instead?

Reply:
Is there one for charts?

Reply:
I think @markstory should know. I think so?

Reply:
We have been using the `Placeholder` component to reserve space for charts/tables to reduce the number of spinners on the page at a single time. It could be useful here as `Placeholder` can also be sized with props (instead of emotion) to prevent layout shifts.

Reply:
I'll follow up on this in https://github.com/getsentry/sentry/pull/25211

Reply:
Fixed in https://github.com/getsentry/sentry/pull/25211/commits/b7f471e740a5dff2f74c35866530f901a0308ef7